backend/provisioner/registry.go

```diff
@@ -273,9 +276,18 @@ func ExtractResources(msg *ftlv1.CreateDeploymentRequest) (*ResourceGraph, error
 			to:   dep.ResourceId,
 		})
 	}
-
+	runnerResource := &provisioner.Resource{
+		ResourceId: module.GetName() + "-runner",
```

Comment:
Should we use deployment id here instead? I think that would fit better to the given model, given that each deployment has a separate runner as a resource.

Reply:
Good point

Reply:
Actually we don't really seem to have the deployment name here?

Reply:
Hmm, yeah. I think it currently created after the provisioning, which is annoying. I suppose that is fine for now, though it breaks the model a bit.